aries-site/src/data/structure.js

```diff
@@ -399,12 +441,19 @@ export const structure = [
     seoDescription:
       'MaskedInput allows you to specify formailzed text within a form field.',
     sections: [],
+    previewComponent:{
+      component: () => <MaskedInputPreview />,
+      cardColor: 'background-front',
+    },
   },
   {
     name: 'Menu',
     description: 'Menu is used to filter or sort content on a page.',
     seoDescription: 'Menu is used to filter or sort content on a page.',
     sections: [],
+    previewComponent: {
+      component: () => <MenuPreview />,
```

Comment:
Not sure if this will be helpful or not, but what if ContentCard checked for the `typeof` previewComponent. If it's an object, then it functions as you currently have it set up. But, if it's just a function it just renders the component. That way, for previewComponents with no extra styling, you could just have `previewComponent: () => <MenuPreview />`

Soft opinion here.

Reply:
But if we lean towards having the default background be background front, then it might save some code.